NetKAN/ProceduralFairings-AutoShroudRemover.netkan

```diff
@@ -0,0 +1,19 @@
+{
+    "spec_version"   : "v1.4",
+    "$kref"          : "#/ckan/github/Felger/FelgerFixes",
+    "ksp_version"    : "1.0",
+    "name"           : "Procedural Fairings - Auto-Shroud Remover",
+    "identifier"     : "ProceduralFairings-AutoShroudRemover",
+    "abstract"       : "Optional ModuleManager patch by Felger to remove the auto-shrouds that spawn when engines are attached to parts, to allow for a more clean use of the Procedural Interstage Fairings.",
+    "license"        : "CC-BY-SA",
+    "release_status" : "stable",
+    "depends" : [
+        { "name" : "ModuleManager" }
+    ],
+    "install" : [
+        {
+            "find"       : "EngineShroudRemover.cfg",
```

Comment:
`find` doesn't work on files, only directories (although I see no reason why it shouldn't also work on files). You'll need:
```
"file"       : "GameData/ZZZ_FelgerFixes/EngineShroudRemover.cfg",
```